Boards.h

```diff
@@ -780,6 +780,25 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_SERVO(p)         (p)
 #define DEFAULT_PWM_RESOLUTION  10
 
+// Adafruit nrf52 boards
+#elif defined(ARDUINO_NRF52_ADAFRUIT)
+#define TOTAL_ANALOG_PINS       NUM_ANALOG_INPUTS
+#define TOTAL_PINS              32
+#define VERSION_BLINK_PIN       LED_BUILTIN
+#define IS_PIN_DIGITAL(p)       ((p) >= 2 && (p) < TOTAL_PINS)
+#define IS_PIN_ANALOG(p)        ((p) == PIN_A0 || (p) == PIN_A1 || (p) == PIN_A2  || (p) == PIN_A3 || \
```

Comment:
kind of. They are 2-5, 28-31. But we have 2 boards with different order of An.
- feather nrf52:  2 (A0), 3, 4, 5, 28, 29, 30, 31 ( A7 )
- metro nrf52 :   30 (A0), 31, 2, 3, 4 ,5 (A5)

There could be more in the future. But we will update the macro